### PR TITLE
Swapped the position of username and password in PSImportSourceCredentials

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/ContainerRegistry/ContainerRegistry/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Swapped the position of username and password in PSImportSourceCredentials [#14971]
```
Comment (dingmeng-xue) [inline src/ContainerRegistry/ContainerRegistry/ChangeLog.md:21]:
```suggestion
* Fixed username and password issue in `Import-AzContainerRegistryImage` [#14971]
```

File: src/ContainerRegistry/ContainerRegistry/Models/PSImportSourceCredentials.cs [hunk 2]
```diff
@@ -22,7 +22,7 @@ namespace Microsoft.Azure.Commands.ContainerRegistry.Models
 {
     public class PSImportSourceCredentials
     {
-        public PSImportSourceCredentials(string password, string username = default(string))
+        public PSImportSourceCredentials(string username, string password)
```
Comment (dingmeng-xue) [inline src/ContainerRegistry/ContainerRegistry/Models/PSImportSourceCredentials.cs:25]: This change makes more sense but it raises compatibility issue. 
Please check if any other method calls this method. If no, this change is safe.

Comment (BethanyZhou) [reply src/ContainerRegistry/ContainerRegistry/Models/PSImportSourceCredentials.cs:25]: Only one method calls this method. It's safe.

**Review by dingmeng-xue — APPROVED** [review]

**Pull request merged**